neo/SmartContract/Native/NativeContract.cs

```diff
@@ -22,7 +22,16 @@ public abstract class NativeContract
         public uint ServiceHash { get; }
         public byte[] Script { get; }
         public UInt160 Hash { get; }
-        public virtual ContractPropertyState Properties => ContractPropertyState.NoProperty;
+        public virtual ContractManifest Manifest => new ContractManifest()
+        {
+            Hash = Hash,
+            Features = ContractPropertyState.NoProperty,
+            Abi = null,
```

Comment:
Should set the abi.

Reply:
The `return` for main is `object`, but we don't have this in the enum, should be an `array` or create a new one in the enum?

Reply:
https://github.com/neo-project/proposals/blob/master/nep-3.mediawiki#parametertype

We have `Any`.

Reply:
https://github.com/neo-project/proposals/blob/master/nep-3.mediawiki#entrypoint

And this is the default abi for all contracts.

Reply:
But what is the value of any ? https://docs.neo.org/en-us/sc/Parameter.html